Formula/immudb.rb

```diff
@@ -19,7 +19,7 @@ class Immudb < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "64b809adca6510ce7879397985ca33f8e1de676bb5453f594b0b753314cb2483"
   end
 
-  depends_on "go" => :build
```

Comment:
Any upstream issue reference for the go1.19 support?

Reply:
While we're using go1.18 to build official binaries, our CI pipeline builds and run tests using go1.19 as well.

Reply:
Why go back to go 1.16 then?

Reply:
thanks for the review. It was an unexpected change when bumping the formula, it seems the local repo had that change

Reply:
That makes total sense. Thanks!!